check/tests/support/mod.rs

```diff
@@ -1,66 +1,64 @@
-extern crate env_logger;
+#![allow(dead_code)]
```

Comment:
I think I'm going to revert this change - I don't like global allows in general...